toqito/channel_props/is_extremal.py

```diff
@@ -0,0 +1,64 @@
+"""Determines whether a quantum channel is extremal."""
```

Comment:
You will also want to add this function to the `__init__.py` file in the root of the `channel_props/` folder.